StyleCop.Analyzers/StyleCop.Analyzers.Test/Verifiers/DiagnosticVerifier.cs

```diff
@@ -23,10 +23,7 @@ public abstract partial class DiagnosticVerifier
         /// <returns>
         /// A new instance of the C# analyzer being tested.
         /// </returns>
-        protected virtual DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
-        {
-            return null;
-        }
+        protected abstract DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer();
```

Comment:
@AArnott The analyzers that don't have a code fix should technically extend `DiagnosticVerifier`, not `CodeFixVerifier`.